runtime/Pervasives.res

```diff
@@ -1,6 +1,6 @@
 /**
    Since [others] depend on this file, its public mli files **should not
-   export types** introduced here, otherwise it would cause 
+   export types** introduced here, otherwise it would cause
```

Comment:
This whole comment is obsolete and should be removed as there is no `others` folder anymore (leftover from the old stdlib build procedure).

Or actually, there should be a module-level doc comment (`/***`) instead explaining what the Pervasives module is.

Reply:
Comment removed https://github.com/rescript-lang/rescript/pull/7346/commits/a36b71d1e699032411c417d4def7fe3ff69aa06a